src/mlpack/core/tree/hrectbound_impl.hpp

```diff
@@ -103,6 +103,27 @@ inline HRectBound<MetricType, ElemType>::HRectBound(
   other.minWidth = 0.0;
 }
 
+/**
+ * Move assignment operator.
+ */
+template<typename MetricType, typename ElemType>
+inline HRectBound<
+    MetricType,
+    ElemType>& HRectBound<MetricType,
+    ElemType>::operator=(HRectBound<MetricType, ElemType>&& other)
```

Comment:
```suggestion
inline HRectBound<MetricType, ElemType>&
HRectBound<MetricType, ElemType>::operator=(
    HRectBound<MetricType, ElemType>&& other)
```
Just a different suggestion for wrapping; I think it's a bit cleaner, but up to you. :+1: